src/scripts/Lookup.js

```diff
@@ -31,42 +36,68 @@ class LookupSelection extends Component {
     }
   }
 
+  pillRef(ref) {
+    this.pill = ref;
+  }
+
+  renderPillResetButton() {
+    return (<Button
+      className='slds-pill__remove'
+      type='icon-bare'
+      icon='close'
+      alt='Remove'
+      tabIndex={ -1 }
+      onClick={ this.props.onResetSelection }
+      key={'resetButton'}
+    />);
+  }
+
+  renderPillSelectedLabel(selected, htmlAttributes) {
+    return (<span
+      className='slds-pill__label'
+      {...htmlAttributes}
+      key={'selectedLabel'}
+    >{ selected.label }</span>);
+  }
+
+  renderPillSelectedIcon(selected) {
+    return (selected.icon ?
+      <Icon
+        className='slds-pill__icon'
+        category={ selected.category }
+        icon={ selected.icon }
+        key={'selectedIcon'}
+      /> :
+      undefined);
+  }
+
+  renderLookUpData(selected, htmlAttributes) {
```

Comment:
don't pass props to render functions
`renderLookUpData() { const { selected, htmlAttributes } = this.props...`